start-docker-nix-build-slave

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Requires: curl, docker, grep
+
+working_dir_name=".nix-docker-build-slave"
+working_dir="$HOME/.nix-docker-build-slave"
+mkdir -p "$working_dir"
+
+ssh_id_file="$working_dir/insecure_rsa"
+remote_sys_conf="$working_dir/remote-systems.conf"
+
+ssh_config="$HOME/.ssh/config"
+
+docker_machine_name="nix-docker-build-slave"
+
+# -- Display info and troubleshooting tips --
+echo "## Use Docker Machine as Nix Build Slave"
+echo "##"
+echo "## Usage: source $0"
+echo "##"
+echo "## Note: if you experience issues, you can clean up the build slave artifacts and start over by:"
+echo "##   1. rm -r ~./$working_dir_name"
+echo "##   2. Delete entry in ~/.ssh/config for"
+echo "##        Host \"$docker_machine_name\"" 
+echo "##   3. Delete the docker container named"
+echo "##        $docker_machine_name"
+echo
+
+# -- Download SSH credentials for docker machine --
+echo ">>> Downloading SSH credentials for the docker machine"
+rm -f "$ssh_id_file" "$ssh_id_file.pub"
+curl https://raw.githubusercontent.com/LnL7/nix-docker/master/ssh/insecure_rsa > "$ssh_id_file"
+curl https://raw.githubusercontent.com/LnL7/nix-docker/master/ssh/insecure_rsa.pub > "$ssh_id_file.pub"
```

Comment:
Since this is part of the repo now, it could conceivably use these from the local repo. However, as is, this script can be downloaded with curl and run. That might be a pretty desirable feature.